tools/docker/tests/test_flows.sh

```diff
@@ -21,9 +21,13 @@ error=0
 start_gw_repeater() {
     dbg "delete running containers before starting test"
     eval docker rm -f gateway $redirect
-    eval docker rm -f repeater $redirect
-    dbg "start gw and repeater"
-    eval ${scriptdir}/test_gw_repeater.sh -d 5 $redirect
+    eval docker rm -f repeater $redirect    
```

Comment:
This is not needed anymore since the `-f` argument to `test_gw_repeater.sh` will remove the repeater.

Reply:
Ok